service/Dockerfile

```diff
@@ -61,8 +59,8 @@ RUN echo yes | keytool -import -file /opt/ldapcert.pem -alias ldaps -keystore $J
 FROM base AS base-ssl-true
 ENV SSL_ENABLED=true
 RUN echo "This stage sets SSL=$SSL_ENABLED"
-# Copy Tomcat properties (to enable SSL)
-COPY ./src/main/resources/tomcat-ssl/server.xml /usr/local/tomcat/conf/
+# Enable SSL: Add our SSL connector after <Service name="Catalina"> is found in server.xml; .bak = backup
+RUN sed -i.bak '/<Service name="Catalina">/a <Connector port="8443" protocol="HTTP/1.1" SSLEnabled="true" maxThreads="150" scheme="https" secure="true" keystoreFile="/usr/local/tomcat/conf/selfsigned.p12" keystorePass="changeit" clientAuth="false" acceptCount="100"/>' /usr/local/tomcat/conf/server.xml
```

Comment:
In case you were wondering on how this is done, the method used is this:
Basically, `sed` does not only replace stuff (most common usage) but it can also append things after a match using the `a` command:
`sed '/^anothervalue=.*/a after=me' test.txt` 

[src](https://fabianlee.org/2018/10/28/linux-using-sed-to-insert-lines-before-or-after-a-match/)